lib/logstorage/syslog_parser.go

```diff
@@ -94,7 +94,7 @@ func (p *SyslogParser) addField(name, value string) {
 // Parse parses syslog message from s into p.Fields.
 //
 // p.Fields is valid until s is modified or p state is changed.
-func (p *SyslogParser) Parse(s string) {
+func (p *SyslogParser) Parse(s string, severityAsLevel bool) {
```

Comment:
I'm not a fan of adding extra boolean arguments, as this approach doesn't scale well. What do you think about adding it as a property of the SyslogParser instead?